docs/01-overview/main-areas/eventing/evnt-02-useful-links.md

```diff
@@ -7,16 +7,16 @@ Find out more about Eventing by following the links on this page.
 To learn more about how Eventing works, see:
 
 - [Eventing architecture](../../../05-technical-reference/00-architecture/evnt-01-architecture.md) - describes how Eventing works and the main actors involved, such as the Eventing Controller and Event Publisher Proxy. You can also find more information about event types here.
-- [Event processing and delivery](../../../05-technical-reference/evnt-01-event-processing.md) - contains a diagram and explanation of the the Eventing processing and delivery flow.
+- [Event types](../../../05-technical-reference/evnt-01-event-types.md) - contains a diagram and explanation of the the Eventing processing and delivery flow.
```

Comment:
The target doc `evnt-01-event-types.md` does not contain any diagram and its about event types, not the Eventing processing and delivery flow, that is described in `evnt-01-architecture.md `

Reply:
I changed the doc but forgot the description 🤦‍♀️ Thank you!